src/__tests__/login.test.ts

```diff
@@ -58,12 +58,10 @@ describe('login', () => {
       password: 'paswword',
     });
 
-    expect(res.statusCode).toBe(200);
+    expect(res.statusCode).toBe(500);
```

Comment:
should it still error with the new selectors? don't we want to handle it?

Reply:
yes but it's a different error, the selectors works fine just there are no cookies sent back, which was an untested error so I kill to bird with one stone. I can add a specific test where everything goes well if you want, or more weird input scenarios

Reply:
oh oki! as you want it's mostly to ensure we did not broke the current behavior